ci/prow/testgrid_config.go

```diff
@@ -67,8 +67,12 @@ var (
 // baseTestgridTemplateData contains basic data about the testgrid config file.
 // TODO(Fredy-Z): remove this structure and use baseProwJobTemplateData instead
 type baseTestgridTemplateData struct {
-	TestGroupName string
-	Year          int
+	ProwHost          string
+	TestGridHost      string
+	GubernatorHost    string
```

Comment:
Seems like testgrid only uses `ProwHost`, are the other 3 newly added vars included just for consistency?

Reply:
Yes.